public/index.html

```diff
@@ -16,17 +16,11 @@
     <link rel="icon" type="image/png" href="%PUBLIC_URL%/favicon.png" integrity="%REACT_APP_SRI_FAVICON_PNG%" crossorigin="anonymous" />
     <link rel="icon" type="image/svg+xml" href="%PUBLIC_URL%/favicon.svg" integrity="%REACT_APP_SRI_FAVICON_SVG%" crossorigin="anonymous" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no,viewport-fit=cover" />
-    <meta property="og:title" content="ShapeShift DAO">
     <meta property="og:type" content="website" />
-    <meta property="og:description" content="Your Web3 & DeFi Portal">
```

Comment:
this will break link previews for discord/twitter etc, as long as you're aware of that

Reply:
nvm i see we're doing this below in the seo component

Reply:
Yah I guess we can leave this in for when they require SSR